docs/upgrade.md

```diff
@@ -44,3 +44,25 @@ $ kubectl krew upgrade directpv
 
 ### Upgrade of release binary
 Refer to the [binary installation documentation](./installation.md#installation-of-release-binary).
+
+## Deprecations
```

Comment:
1. I think we don't need to keep this in the documentation of `v4.1`. This should go into release notes.
2. The release note should indicate `PodSecurityPolicy` removal (not deprecation) as per Kubernetes.
3. We do not need to document how to backup/apply `PodSecurityPolicy` anywhere as it is complete removal from DirectPV.
4. We would need to fix `docs/tools/install.sh` script as well by aborting it if `PodSecurityPolicy` used in case of upgrade.

Reply:
`docs/tools/install.sh` will use `kubectl directpv install --declarative` which will  not generate PSP manifests @balamurugana

Reply:
In case of upgrade, `install.sh` will upgrade everything except previously created PSP. As we are removing PSP support completely, we would error out for the case i.e. we fail to upgrade with a message saying use the latest version of `v4.0`